carball/analysis/stats/tendencies/averages.py

```diff
@@ -1,26 +1,23 @@
-from typing import List
-
+from typing import List, Dict
 import pandas as pd
+from ....analysis.stats.stats import BaseStat
 
 from ....generated.api import game_pb2
 from ....generated.api.player_pb2 import Player
 from ....generated.api.stats.events_pb2 import Hit
+from ....generated.api.stats.player_stats_pb2 import PlayerStats
+from ....json_parser.game import Game
+
 
+class Averages(BaseStat):
+    def calculate_player_stat(self, player_stat_map: Dict[str, PlayerStats], game: Game, proto_game: game_pb2.Game,
```

Comment:
Did this not used to work? (I'm assuming calculate_player_stat is being called from some place higher up and I'm thus wondering what used to call `get_averages_for_player`)

Reply:
This was just separating averages to its own file